### ISSUE TITLE
can't connect when link not specified

### ISSUE THREAD

**Issue by Morriz** [mongo-express/mongo-express#196]
Hi, I don't need container linking, since I do service discovery. Now if I just run
```
docker run -it --rm -e ME_CONFIG_MONGODB_SERVER="mongo" mongo-express
```
I get the following error:
```
Error: getaddrinfo ENOTFOUND mongo mongo:27017
    at errnoException (dns.js:26:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:77:26)
```
mongo:27017 is accessible, and other containers make use of the same connection...

Is there anything going on that breaks lookups by hostname?

**Comment by Morriz** [comment]
Nevermind, I was going cross namespace in kubernetes...doesn't work

**Comment by banomaster** [comment]
Hey @Morriz,

I just came across this question, where i found out, that you probably exposed mongo-express as a microservice with kubernetes, where you connected to mongo pod via kubernetes namespaces.

I have three kubernetes services for three kubernetes with namespaces mongo-1, mongo-2, mongo-3 like it is described [here](https://medium.com/google-cloud/mongodb-replica-sets-with-kubernetes-d96606bd9474#.jia44y2fz).

As host in mongo-express I set "mongo-1,mongo-2,mongo-3" and as the url 'mongodb://mongo-1,mongo-2,mongo-3:27017' and I am getting the same error as you did.

Maybe you have some advice for me how to get rid of it or what I am doing wrong here.

Thank you in advance!
